test/contrib/bigquery_gcloud_test.py

```diff
@@ -27,23 +27,30 @@
 
 import luigi
 import unittest
-from luigi.contrib import bigquery, gcs
 
 try:
     import googleapiclient.errors
     import oauth2client
 except ImportError:
     raise unittest.SkipTest('Unable to load googleapiclient module')
+from luigi.contrib import bigquery, bigquery_avro, gcs
+import avro.schema
+from avro.datafile import DataFileWriter
+from avro.io import DatumWriter
 
+from contrib import gcs_test
 from nose.plugins.attrib import attr
+from helpers import unittest
 
 # In order to run this test, you should set your GCS/BigQuery project/bucket.
 # Unfortunately there's no mock
-PROJECT_ID = os.environ.get('BQ_TEST_PROJECT_ID', 'your_project_id_here')
-BUCKET_NAME = os.environ.get('BQ_TEST_INPUT_BUCKET', 'your_test_bucket_here')
+PROJECT_ID = gcs_test.PROJECT_ID
```

Comment:
Commit https://github.com/spotify/luigi/pull/1917/commits/a5291aba437c796a1a3ba080b13570bf5149e275, which introduced `BQ_` things, and which you rebased upon, had explicit intent to separate BigQuery test modules from `gcs_test` module. The resolution to include `gcs_test` again reverses it. Any good reason to couple them?